react/features/base/conference/middleware.js

```diff
@@ -3,12 +3,11 @@
 import UIEvents from '../../../../service/UI/UIEvents';
 
 import {
-    _LOCAL,
-    _REMOTE,
-    AUDIO_ONLY_DISABLED,
-    PINNED_,
-    UNPINNED_,
-    sendAnalyticsEvent
+    ACTION_PINNED,
+    ACTION_UNPINNED,
```

Comment:
What was your decision making for when to leave an event name as a constant and when to inline the name as a parameter to an event factory call?